packages/smooth_app/lib/background/abstract_background_task.dart

```diff
@@ -116,4 +113,15 @@ abstract class AbstractBackgroundTask {
     }
     return stringBuffer.toString();
   }
+
+  static void showSnackBar(
```

Comment:
Shouldn't this method be moved in the helper part of the app?
If I remember well, we often use a Snackbar

Reply:
The point there was to have all _background task-related_ snackbars look the same, not all the app snackbars.